app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PostsController < ApplicationController
```

Comment:
The example you followed for file uploads had posts but your app doesn't have posts remove this controller.